packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,560 @@
+{
+	"enable_central_scene": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Central Scene",
+		"defaultValue": 1
+	},
+	"multilevel_enable": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"valueSize": 4,
+		"defaultValue": 1
+	},
+	"multilevel_upper_function": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Upper switch value",
+		"valueSize": 4,
+		"defaultValue": 255
```

Comment:
255 doesn't seem compatible with the imported parameter that has a range of 0 to 99

Reply:
the upper value is 255 but value range is from 0-99 ?
So im not sure if this should be changed.
![image](https://user-images.githubusercontent.com/33527165/114500740-f819d700-9c28-11eb-8555-f73f75fbb209.png)

Reply:
added this: "description": "Allowable range: 0-99",
based on the style guide